pkg/kubelet/cm/dra/manager.go

```diff
@@ -68,22 +68,35 @@ func NewManagerImpl(kubeClient clientset.Interface, stateFileDirectory string) (
 func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 	for i := range pod.Spec.ResourceClaims {
 		podClaim := &pod.Spec.ResourceClaims[i]
-		claimName := resourceclaim.Name(pod, podClaim)
-		klog.V(3).InfoS("Processing resource", "claim", claimName, "pod", pod.Name)
+		klog.V(3).InfoS("Processing resource", "podClaim", podClaim.Name, "pod", pod.Name)
+		claimName, mustCheckOwner, err := resourceclaim.Name(pod, podClaim)
+		if err != nil {
+			return fmt.Errorf("prepare resource claim: %v", err)
+		}
 
+		if claimName == nil {
+			// Nothing to do.
+			continue
+		}
```

Comment:
Can we add a comment such as the following here? Looks like it will need to be propagated to 3 places in total.
```
// The claim name might be nil if no underlying resource claim
// was generated for the referenced claim. There are valid use
// cases when this might happen, so we simply skip it.
```

Reply:
[Changed](https://github.com/kubernetes/kubernetes/compare/aff83426e547e83261425b68f50f27356df7a019..fec25785ee6cef3998d2eeb28df068370a9938f2) - please take another look.